### PR TITLE
1329 map collapse

### LINKED ISSUE #1329: have map collapse on medium screens.
Screenshot 2024-08-30 at 3 00 58 PM
Screenshot 2024-08-30 at 3 00 49 PM

### PR DESCRIPTION
**Pull request by dnoneill**
closes #1329
Screenshot 2024-08-30 at 3 14 19 PM
Screenshot 2024-08-30 at 3 14 13 PM
Screenshot 2024-08-30 at 3 13 56 PM
Screenshot 2024-08-30 at 3 12 49 PM

### CODE REVIEW

File: app/assets/stylesheets/earthworks.css [hunk 1]
```diff
@@ -183,12 +183,8 @@ Uncomment lines 142-144
 } */
 
 #main-container .leaflet-container {
-  height: 620px;
-}
-
-@media (max-width: 575.98px) {
-  #main-container #leaflet-viewer  {
-    display: none;
+  @media (min-width: 991.98px) {
```
Comment (marlo-longley) [inline app/assets/stylesheets/earthworks.css:186]: Only need to do the 0.02 thing for max-width. For min-width, use 992.

**Comment by marlo-longley** [comment]
This is great. Any chance we can get the button to look more like the Figma?
Screenshot 2024-08-30 at 15 58 05
vs.
Screenshot 2024-08-30 at 15 58 51

**Comment by dnoneill** [comment]
Screenshot 2024-08-30 at 4 10 13 PM

Updated view. Good catch.

**Comment by marlo-longley** [comment]
Does the Search using map button have a toggled on/off state? As in, if you click it and open the map, should the button change style at all? Currently it does not.

**Comment by dnoneill** [comment]
I didn't see one in the figma and the facets don't have one in figma so I figured we could update later it if we heard it has a state from Darcy.

**Comment by jcoyne** [comment]
@dnoneill this needs a rebase

**Comment by dnoneill** [comment]
@jcoyne Done.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**